anatomy/track-tooling/analyzers/creating-from-scratch.md

```diff
@@ -6,6 +6,8 @@ These are the steps to get going:
 
 1. Check [our repository list for an existing `...-analyzer`](https://github.com/exercism?q=-analyzer) to ensure that one doesn't already exist.
 2. Scan the [contents of this directory](./) to ensure you are comfortable with the idea of creating an Analyzer.
-3. Open an issue at [exercism/exercism](https://github.com/exercism/exercism/issues/new) introducing yourself and telling us which language you'd like to create a Analayzer for.
+3. Open an issue at [exercism/exercism][exercism-repo] introducing yourself and telling us which language you'd like to create a Analayzer for.
```

Comment:
Why not do:

```suggestion
3. [Open an issue at exercism/exercism](https://github.com/exercism/exercism) introducing yourself and telling us which language you'd like to create a Analyzer for.
```

Reply:
Two reasons:
1. Your other PR said we shouldn't use internal links, so I moved it to the end of the file 😛 
2. I figured we might have more info at the apex of the exercism/exercism directory, and possibly have issue templates and things so that it might be better to send someone to the apex. But I can be persuaded either way on this.

Reply:
1. True :)
2. It would be really nice to have an issue template. That is something we should do at some point. Feel free to merge as is. I'll approve.

Reply:
Ta :)